src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,42 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE_FOR_CANDIDATE = 35;
+    private static final int MIN_PERIODS_IN_UKRAINE = 10;
+    private static final String NATIONALITY = "Ukrainian";
+    private static final String DELIMITER = "-";
+
+    public boolean checkTimeLivingInCountry(Candidate candidate) {
```

Comment:
```suggestion
    private boolean checkTimeLivingInCountry(Candidate candidate) {
```

after all public methods